src/ontology/uberon-edit.obo

```diff
@@ -222488,6 +222488,14 @@ relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500109
+name: basion
+def: "An anatomical point on the ventral border of foramen magnum in the median plane." [https://www.sciencedirect.com/topics/veterinary-science-and-veterinary-medicine/basion]
```

Comment:
The dbxref seems to a list of search results that is subject to change. 
"These pages are auto-generated by ScienceDirect using heuristic and machine-learning approaches to extract relevant information from our extensive collection of content. We compile this information on a topic-by-topic basis providing the reader both depth and breadth on a specific area of interest."

Can you find a stable reference to support this definition?